### PR TITLE
Update governments.yml

### PR DESCRIPTION
**Pull request by adam-moss**
Thanks for the contribution! **Below** is a template that can make it easier when submitting your Organization:

**Your Organization**: Department for Work and Pensions  
**GitHub Organization url**: @dwp
**Country/Locality**: London, United Kingdon

Inclusion requirements:
- [X] Referenced account is an [Organization](https://github.com/github/government.github.com#add-organization) not a User^
- [X] If this is a government project, your Organization's description should include a reference to your country/agency.
- [X] Make sure your Organization includes at least one public repository
- [X] Please also include a URL for your organization that links back to your organization or agency homepage.
 
^ If you want to convert your account to an org, details can be found here: https://help.github.com/articles/converting-a-user-into-an-organization/
  
 :heart: GitHub Government Contributors

### CODE REVIEW

File: _data/governments.yml [hunk 1]
```diff
@@ -410,14 +410,14 @@ U.K. Central:
   - datagovuk
   - decc
   - defra
-  - Department-for-Work-and-Pensions
```
Comment (nschonni) [inline _data/governments.yml]: I looks like this org still exists, but just hasn't been active

Comment (adam-moss) [reply _data/governments.yml]: Yes, we are looking to merge that into our core dwp org at the moment.

Comment (adam-moss) [reply _data/governments.yml]: Right, I've merged that into our core org now and removed the Department-for-Work-and-Pensions one so we're good to go I think.

**Comment by jbjonesjr** [comment]
thanks for this @adam-moss

**Pull request merged**